### PR TITLE
Create .appmanifest file for PWA

### PR DESCRIPTION
**Pull request by ghost**
Also includes some fallback fixes for PWA optimisation.

### CODE REVIEW

File: app/public/app.webmanifest [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "lang": "en",
```
Comment (kearfy) [inline app/public/app.webmanifest:2]: Maybe you can figure out how we can add a lang="en" attribute to the html element with next aswell, I think Lighthouse is complaining about that aswell

File: app/public/app.webmanifest [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "dir": "ltr",
+
+    "name": "Kards",
+    "short_name": "Kards",
+    "id": "kards",
+    "description": "Socially social: Kards.",
+
+    "icons": [{
+        "src": "LogoSmall.svg",
```
Comment (kearfy) [inline app/public/app.webmanifest]: Do android and ios both "like" svg's? png might be a saver option but could be wrong.

**Review by kearfy — CHANGES_REQUESTED** [review]

File: app/public/app.webmanifest (hunk 2 above)
Comment (ghost) [reply app/public/app.webmanifest]: PNGs would require multiple files with separate sizes, unless there is some resizing algorithm in Next than can warrant grabbing resized images automatically. If those options are available, would require some more implementation.

File: app/public/app.webmanifest (hunk 1 above)
Comment (ghost) [reply app/public/app.webmanifest:2]: Resolved in latest commit.

File: app/public/app.webmanifest (hunk 2 above)
Comment (kearfy) [reply app/public/app.webmanifest]: Well, if SVG's are supported everywhere, then that would be my preference as it's simply smaller in size. Just wondering about that though

**Review by kearfy — CHANGES_REQUESTED** [review]

File: app/public/app.webmanifest (hunk 2 above)
Comment (ghost) [reply app/public/app.webmanifest]: For the sake of safety, added PNG icons.

**Review by kearfy — APPROVED** [review]

**Pull request merged**